packages/pn-personafisica-login/src/pages/login/SpidSelect.tsx

```diff
@@ -23,13 +23,22 @@ const SpidSelect = ({ onBack }: { onBack: () => void }) => {
   const shuffledIDPS = shuffleList(IDPS.identityProviders);
 
   const getSPID = (IDP: IdentityProvider) => {
+    sessionStorage.setItem('IDP', IDP.entityId);
```

Comment:
We never remove the element stored in the sessionStorage. We should remove on login success and failure